persistence-modules/hibernate-parameters/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>hibernate-parameters</artifactId>
+    <name>Hibernate Named Parameters Example</name>
```

Comment:
```suggestion
    <name>hibernate-parameters</name>
```

Reply:
guidelines state that the name tag is optional but if provided, must match artifactId

Reply:
Let's also add the <packaging>jar</packaging> element